src/shell-js.cpp

```diff
@@ -61,14 +61,15 @@ shell_js_add_extension_importer (const char  *target_object_script,
       goto out;
     }
 
-  if (!JSVAL_IS_OBJECT (target_object))
+  if (!target_object.isObjectOrNull ())
```

Comment:
Well, this is exactly the equivalent of the code that was there before; `JSVAL_IS_OBJECT` and `..._TO_OBJECT` confusingly counted `null` as an object. I can definitely see changing it to `isObject()` and `toObject()`, but I thought I'd err on the side of not changing the behaviour.

Now that I think about it, though, it doesn't make sense to define an importer on `null` since `null` can't have properties.